src/ripple/protocol/TxFlags.h

```diff
@@ -127,27 +127,14 @@ constexpr std::uint32_t tfPayChanClaimMask = ~(tfUniversal | tfRenew | tfClose);
 // NFTokenMint flags:
 constexpr std::uint32_t const tfBurnable                   = 0x00000001;
 constexpr std::uint32_t const tfOnlyXRP                    = 0x00000002;
-constexpr std::uint32_t const tfTrustLine                  = 0x00000004;
 constexpr std::uint32_t const tfTransferable               = 0x00000008;
+constexpr std::uint32_t const tfMutable                    = 0x00000016;
```

Comment:
Flags are bitmasks, they should only be one bit and the number is already in hex.
```suggestion
constexpr std::uint32_t const tfMutable                    = 0x00000010;
```